patches/openwrt/0026-ar71xx-Add-support-for-TP-Link-WBS210-v2.patch

```diff
@@ -0,0 +1,148 @@
+From fa599cbdc721b445e6909b75af82620619cb6bce Mon Sep 17 00:00:00 2001
```

Comment:
This patch needs to go upstream to OpenWrt first. Unfortunately they will not take new devices for ar71xx, so you would need to port it to ath79 first.

Reply:
Think you've done it, it's now upstream. We'll add the ath79-generic target in #1848. After that is in we can take a backport of this commit.

https://github.com/openwrt/openwrt/commit/3930aab2cb067d04f5c9ebbf1af57744e19355eb